clash_royale_service.py

```diff
@@ -257,6 +257,43 @@ def clan_remaining_war_attacks(self, clan_tag: str) -> List[ClanRemainingWarAtta
                     ),
                 )
             )
-
         all_clan_attacks.sort(key=lambda x: (x.medals, x.fame), reverse=True)
         return all_clan_attacks
+
+    def clan_players_remaining_war_attacks(self, clan_tag: str) -> List[PlayersRemainingWarAttacks]:
+        curr_river_race = self.clash_royale_client.get_current_river_race(
+            clan_tag)
+        curr_river_race_participants = {}
+        all_current_war_players = {}
+        all_current_war_players_output = []
+        for participant in curr_river_race['clan']['participants']:

```

Comment:
I'm confused, doesn't participants include people currently in the clan as well or no?

Reply:
In separate PR, lets have default behaviour be to exclude randoms no longer in clan and users must specify argument `i` to include them.

We should be consistent with ausclanremaining which already does that (i.e. if a player is not in the clan but used 1 deck, we treat them as having used up 4 decks instead, they're dead to us, why do we care to track them anymore or not?)

Reply:
done